code/modules/surgery/organs/augments_chest.dm

```diff
@@ -175,6 +175,12 @@
 
 	allow_thrust(THRUST_REQUIREMENT_SPACEMOVE)
 
+/obj/item/organ/cyberimp/chest/thrusters/proc/spacemove_react(mob/user, movement_dir)
+	SIGNAL_HANDLER
+
+	if(on && movement_dir)
+		return COMSIG_MOVABLE_STOP_SPACEMOVE
```

Comment:
signal isn't registered anywhere

Reply:
it doesnt need to be? It's part of Process_Spacemove()

Reply:
why is it a signal handler then?

Reply:
every proc/move_react() has a SIGNAL_HANDLER

Reply:
actually I think I just forgot

Reply:
05642c1